Extension/src/cpptoolsJsonUtils.ts

```diff
@@ -0,0 +1,95 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
The purpose of the "cpptools.json" file is for A/B testing.  This module would be better named "abTesting.ts" or similar.

Reply:
I have some ideas for a refactor here that I won't burden you with in this review.